paper.md

```diff
@@ -78,8 +78,44 @@ Figures can be included like this: ![Fidgit deposited in figshare.](figshare_art
 
 ## Story of the project
 
-We are a group of graduate students who connected through the [UofT Coders](http://uoftcoders.github.io/) club at the University of Toronto. We created and ran a fourth-year course in the Department of Ecology and Evolutionary Biology titled “Theoretical Ecology and Reproducible Quantitative Methods in R.” We modelled the structure and portions of the course content on “Reproducible Quantitative Methods” a course created by Dr. Christie Bahlai [ ], modifying the lesson content to include more theoretical ecology topics but maintaining the focus on reproducibility and computational skills. Our teaching team was six graduate students from several different fields (Physiology, Biomedical Engineering, Physics, Psychology, and Nutritional Science); we divided course topics among each instructor to develop and deliver individual lessons and assignments.
+While there are many excellent open source libraries for quantitative data
+analysis, the use of subpar analysis tools (such as spreadsheet software) is
```

Comment:
Do we want to make a value judgment on spreadsheet software? Maybe "less efficient and reproducible"

Reply:
Haha, I went back and forth on this like 5 times, between 'inefficient' or 'subpar'. For the reasons you mentioned. Including reproducibility is good, will change in a bit.